modules/developer-cli-odo-installing-odo-on-linux-rpm.adoc

```diff
@@ -0,0 +1,56 @@
+[id="installing-odo-on-linux-rpm_{context}"]
+
+= Installing {odo-title} on {op-system-base-full} using an RPM 
+
+For {op-system-base-full}, you can install the `{odo-title}` CLI as an RPM.
+
+.Procedure
+
+. Register with Red Hat Subscription Manager:
++
+[source,terminal]
+----
+# subscription-manager register
+----
+
+. Pull the latest subscription data:
++
+[source,terminal]
+----
+# subscription-manager refresh
+----
+
+. List the available subscriptions:
++
+[source,terminal]
+----
+# subscription-manager list --available --matches '*OpenShift Developer Tools and Services*'
+----
+
+. In the output of the previous command, find the  `Pool ID` field for your {product-title} subscription and attach the subscription to the registered system:
++
+[source,terminal]
+----
+# subscription-manager attach --pool=<pool_id>
+----
+
+. Enable the repositories required by `{odo-title}`:
++
+[source,terminal]
+----
+# subscription-manager repos --enable="ocp-tools-4.9-for-rhel-8-x86_64-rpms"
+----
+
+. Install the `{odo-title}` package:
++
+[source,terminal]
+----
+# yum install odo
+----
+
+. Verify that `{odo-title}` is now available on your system:
```

Comment:
I like the inclusion of the verification step. Verification steps are often not included but If ind them helpful, good idea including it!